libcob/common.h

```diff
@@ -1025,7 +1025,7 @@ enum cob_file_operation {
 #define COB_STATUS_49_I_O_DENIED			49
 #define COB_STATUS_51_RECORD_LOCKED			51
 #define COB_STATUS_52_DEAD_LOCK				52	/* currently not implemented (patch available) */
-#define COB_STATUS_53_MAX_LOCKS				53	
+#define COB_STATUS_53_MAX_LOCKS				53	/* currently not implemented */
 #define COB_STATUS_54_MAX_LOCKS_FD			54	/* currently not implemented */
```

Comment:
Well it seems implemented (at least for relatives files and indexed files with ISAM backend).

Reply:
I kind of guessed that. It seems this cannot be easily tested (only the deadlock can, using multiple processes).
The please drop that comment for those that are used when doing the merge-commit.

Note: I'm often on-call duty today, so would be able to do my 32bit testsuite fix and the create_mingw_dist.sh merge in a short period of time after your check-in, just in case you want to go on over the weekend [similar tomorrow].

Reply:
> The please drop that comment for those that are used when doing the merge-commit.

Done (also for `COB_STATUS_52_DEAD_LOCK`).
The batch has been committed to SVN.

> Note: I'm often on-call duty today, so would be able to do my 32bit testsuite fix and the create_mingw_dist.sh merge in a short period of time after your check-in, just in case you want to go on over the weekend [similar tomorrow].

Well I don't expect people to work on week-ends, and in general I don't either (well... I try not to, but I always end up doing some work 😅). So yeah, maybe I'll be merging some more commits this weekend, once you're done with your changes ;)